eds/blocks/storyteller/storyteller.css

```diff
@@ -262,6 +293,14 @@
     display: block;
   }
 
+  .storyteller .foreground-container picture, .storyteller p:nth-of-type(1) picture {
+    display: block;
+  }
+
+  .storyteller .foreground-container a, .storyteller picture.hide-poster {
+    display: none;
+  }
```

Comment:
isn't this the same or similar to line 61?

Reply:
Correct. Removed